src/datahike/config.cljc

```diff
@@ -158,17 +165,17 @@
                  :initial-tx (:datahike-intial-tx env)
                  :keep-history? (bool-from-env :datahike-keep-history *default-keep-history?*)
                  :attribute-refs? (bool-from-env :datahike-attribute-refs *default-attribute-refs?*)
-                 :name (:datahike-name env (or *default-db-name* (z/rand-german-mammal)))
```

Comment:
I wrote to you about in January or so. The problem is that I cannot de-duplicate the configuration if it is randomized. If `zufall` could be seeded with a random seed than it would be possible. I did not find dependencies on the `:name` feature so I decided to remove it. Note that I added `store-identity` instead, which fully identifies each database through the store it is based on. It is human readable and informative, but not as fun as `zufall` names.